apps/devtools/app/scripts/components/MessageViewer/MessagesViewer.tsx

```diff
@@ -94,10 +108,35 @@ export const MessagesViewer: React.FC<MessageViewerProps> = ({
           flex={0}
           zIndex={2}
         />
-        <MessageContent
-          message={selectedQueryMessage}
-          isRecording={isRecording}
-        />
+        <Box display="flex" flexDirection="column" width="70vw">
+          <Accordion allowToggle>
+            <AccordionItem>
+              {({ isExpanded }) => (
```

Comment:
Render props? I haven't seen this pattern in quite some time. Just curious why you used that here?

Reply:
Nevermind. I guess that's a chakra-ui thing.

Reply:
Yeah wanted the expanded state here to be able to know when this item is open. Chakra is pretty good about making the state available to hooks, but it seems like this is the best way to get the state via accordions.